app/Http/Controllers/GalleryController.php

```diff
@@ -10,4 +12,20 @@ public function index()
     {
         return view('gallery');
     }
+
+    public function store()
```

Comment:
Arr you sure this belongs here? We are in gallery controller 🤔

Reply:
deleted!